src/Passwordless/PasswordlessClient.cs

```diff
@@ -11,6 +11,11 @@ namespace Passwordless;
 [DebuggerDisplay("{DebuggerToString(),nq}")]
 public class PasswordlessClient : IPasswordlessClient, IDisposable
 {
+    private static readonly string SdkVersion =
+        typeof(PasswordlessClient).Assembly.GetName().Version?.ToString(3) ??
+        // This should never happen, unless the assembly had its metadata trimmed
+        throw new InvalidOperationException("Could not determine SDK version.");
```

Comment:
Can that happen? If so, should we fail on a less hardcore level at set "unknown"?

Reply:
It can happen in theory. You can open the assembly in ILSpy and scrub that data. It shouldn't happen in most sane situations.

My initial motivation for an exception was because if this _does_ happen, we really want to know. But then again, the version isn't that crucial, so it might be better to set it to `unknown`. I'll change that.